config/routes.rb

```diff
@@ -3,4 +3,6 @@
   devise_for :users
   resource :dashboard, only: :show
   resources :habits, only: [:new, :create]
+
+  post '/twilio/reply' => "twilio#reply"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.